index.html

```diff
@@ -1,19 +1,27 @@
+      <img src="data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/4gHYSUNDX1BST0ZJTEUAAQEAAAHIAAAAAAQwAABtbnRyUkdCIFhZWiAH4AABAAEAAAAAAABhY3NwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAA9tYAAQAAAADTLQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAlkZXNjAAAA8AAAACRyWFlaAAABFAAAABRnWFlaAAABKAAAABRiWFlaAAABPAAAABR3dHB0AAABUAAAABRyVFJDAAABZAAAAChnVFJDAAABZAAAAChiVFJDAAABZAAAAChjcHJ0AAABjAAAADxtbHVjAAAAAAAAAAEAAAAMZW5VUwAAAAgAAAAcAHMAUgBHAEJYWVogAAAAAAAAb6IAADj1AAADkFhZWiAAAAAAAABimQAAt4UAABjaWFlaIAAAAAAAACSgAAAPhAAAts9YWVogAAAAAAAA9tYAAQAAAADTLXBhcmEAAAAAAAQAAAACZmYAAPKnAAANWQAAE9AAAApbAAAAAAAAAABtbHVjAAAAAAAAAAEAAAAMZW5VUwAAACAAAAAcAEcAbwBvAGcAbABlACAASQBuAGMALgAgADIAMAAxADb/2wBDABQODxIPDRQSEBIXFRQYHjIhHhwcHj0sLiQySUBMS0dARkVQWnNiUFVtVkVGZIhlbXd7gYKBTmCNl4x9lnN+gXz/2wBDARUXFx4aHjshITt8U0ZTfHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHx8fHz/wAARCAA8AIIDASIAAhEBAxEB/8QAGgABAAMBAQEAAAAAAAAAAAAAAAMEBQIGAf/EADEQAAEEAgECBAQDCQAAAAAAAAEAAgMEBRESITETMkFhBlFx0RQVNCNCUmJyc7HB4f/EABUBAQEAAAAAAAAAAAAAAAAAAAAB/8QAFBEBAAAAAAAAAAAAAAAAAAAAAP/aAAwDAQACEQMRAD8A9giIoCIiAiIgIiICIiAiIginsRVmc5pGsb7lV48tSldxbYaD79FaliZMwslYHtPcEbXm8vhvwzTPW2Yv3m/w/wDEHpx1GwoxPEXcRIzl21yG1gslkq4uGDxHF8o5nZ8rfQBdYqqLE5c8bYzqR8z6IPQIiICIsn4hnlgrRGGR0ZL9EtOvRBrIsiHG3P2bzkZSOhI2VpuniZK2N0jWvcNhpPUhBIigiu1ppDHFPG949AV9ntQVtePKyPfYE9SgmRRwWIbDeUMjZB/Keyyadnw8teM83GNvbk7oOqDaRRQWYbLeUErZAO/E9kntQVgDPK1m+2z3QSouIpo52c4nte35tO1l4eR772QD3ucGyEAE9upQa6+EAggjYPcKnlbT61bcW+bjoHW9KvjrN2y8eJoRt7u4637IKGRINx7W9Gs00D5ALYxdcwVByGnPPIrivjGsmM0x5vJ3r0CvoCIiAsX4n/Sw/wBf+ltKhlaDshCxjZAwtdvZG0EdDHSwSRzOuSSNA8hHTt9VVy0TZ83Tifvi5ujo9xsqZuOyTdD8x6D04qzZoPmyde0HtDYhotPc9/ugzsnViqZKg+uwR836Ib0HQj7q5fsQm2yuKos2NbAOgAPqpMhQfbsVZWva0Qu5EH16j7Li7jpZLjbdScRTAaPIbBQUscHRZ6VhhbByj6sadj0SpWisZ254zA8NJIB7K5Sx1mO+63anY95bx01qjkxlxl2axUssZ4p6hzeyCvAxtX4kdFA3jG5vVo7DptRQSGfK25ZKrrJaeIA1po37/RamPxn4WV880pmsP7u1rSjsYyYW3WqM4hkf5muGwUEWNhniycrxWfBXkb5SRoFML+vyP90/5KvUorcZebc7Zd60Gt1pVm4yeC++xWsBrJXcnscPfaDUREQEREBERAREQEREBERAREQEREBERAREQEREBERB/9k=" alt="">

```

Comment:
these image links are not feasable to be used

Reply:
Yes the section is responsive on all screen sizes

Reply:
use the image links provided , if not there let me know and share screen shots, mobile and web. Thank you.

Reply:
![Screenshot from 2023-10-04 15-33-47](https://github.com/italanta/elewa-website-html/assets/126472845/31a8d83f-c58d-4943-b4a1-95d6d43fd67c)